subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync_iso.c

```diff
@@ -28,6 +28,8 @@
 #include "lll_internal.h"
 #include "lll_tim_internal.h"
 
+#include "ull_internal.h"
+#include "ull_iso_internal.h"
```

Comment:
No, ULL headers are not to be directly included in LLL, but only declare the required functions as `extern`. See for example this: https://github.com/zephyrproject-rtos/zephyr/blob/19601edd9b49446b69bd32bf40ca2da2eee1fbfb/subsys/bluetooth/controller/ll_sw/lll_sync_iso.h#L80

Reply:
Done